girder_style/templates/account/snippets/menu_item.html

```diff
@@ -1,7 +1,9 @@
 <li class="flex flex-grow">
   <a href="{% url url_name %}"
-     class="flex flex-col group justify-center p-4 text-center text-secondary-400 text-xs transition-all w-full hover:text-secondary-300 md:flex-row md:items-center lg:text-sm">
-    <i class="{{ icon_class }} block opacity-75 text-white text-xl transition-all group-hover:opacity-100 md:mr-2"></i>
-    {{ text }}
+     class="flex flex-col group justify-center p-4 text-center text-white text-xs transition-all w-full hover:text-white md:flex-row md:items-center lg:text-sm">
```

Comment:
@jtomeck Could you include a screenshot of how this changes things?

Reply:
<img width="275" alt="Screen Shot 2021-04-27 at 11 28 19 AM" src="https://user-images.githubusercontent.com/1560779/116269686-6cd23280-a74c-11eb-9b89-0818e4f0f017.png">

The light blue was hard to read with a lot of image choices that I tried. So I updated it to make it more legible.